doc/rst/language/spec/tuples.rst

```diff
@@ -872,47 +834,54 @@ For example, in this code:
 
       (0, 0, (x = 0))
 
-Initialization of the tuple variable ``tup`` will make a copy of the
-array ``a``, the record ``r``, and the integer ``i``. Because ``tup`` stores
-a copy of these three variables, changes made to them are not visible
-in ``tup`` when it is written to standard output.
+   Initialization of the tuple variable ``tup`` will make a copy of the
+   array ``a``, the record ``r``, and the integer ``i``. Because ``tup`` stores
+   a copy of these three variables, changes made to them are not visible
+   in ``tup`` when it is written to standard output.
+   Copy-initialization of the tuple's record component would be changed
```

Comment:
With the amount of times you've written this disclaimer, I kind of feel like it might be easier for us to hyperlink to copy-initialization and then specify that copy-initialization may be subject to elision.

Reply:
Done.